save-plugins/fix-plugin/src/commonNonJsTest/kotlin/org/cqfn/save/plugins/fix/FixPluginTest.kt

```diff
@@ -43,6 +43,22 @@ class FixPluginTest {
         assertEquals("Test1Expected.java", pairs.single().second.name)
     }
 
+    @Test
+    fun `tests should have a relative path`() {
```

Comment:
Please, try to adapt this test (add cases with relative paths), so that we will able to see, that in case of `relative path` all is fine, but when we give just name of the file, we return nothing